### DISCUSSION TITLE
For Linux users

### DISCUSSION THREAD

**Discussion by antonreshetov** [massCodeIO/massCode#87] [Announcements]
Unfortunately I cannot test the working capacity in different versions of Linux. 

By default I create a `.snap` package. Which I can test in VM Ubunut 22.04. 

If you need some other build, please do and test it yourself. Download the repo, install the dependencies `pnpm install`, specify the required package in the https://github.com/massCodeIO/massCode/blob/master/config/electron-builder.ts#L27 configuration and run the `pnpm build` for build.

**Comment by decoherencer** [comment]
Don't forget to add` maintainer: 'name'` for pacman build in [electron-builder.ts](https://github.com/massCodeIO/massCode/blob/master/config/electron-builder.ts#L27)

**Comment by StevenJohnRoberts** [comment]
Can you give some advise. I would like to install on my Raspberry Pi which is running Ubuntu Mate.
